scripts/test-build-config.ps1

```diff
@@ -71,16 +71,17 @@ Get-Content -Path "$VCVARSPATH" | Foreach-Object {
 # Add clang binaries to PATH
 $env:PATH += ";C:\Program Files\LLVM\bin"
 
-$BUILD_GENERATOR="Visual Studio 15 2017 Win64"
-$BUILD_ENCLAVES_FLAG=""
-$LINUX_BIN_FLAG="-DLINUX_BIN_DIR=`"$LINUX_BIN_DIR`""
-
-if ($BUILD_ENCLAVES) {
+if ($CROSSCOMPILED_ENCLAVES) {
+    $BUILD_GENERATOR="Visual Studio 15 2017 Win64"
+    $BUILD_ENCLAVES_FLAG="-DBUILD_ENCLAVES=0"
```

Comment:
Sorry to nit-pick, but can you please change `=0` to `=OFF`? Just trying to keep the CMake style consistent, thank you 😄